cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java

```diff
@@ -162,12 +179,32 @@ private long nettyHeapMemory() {
     return -1;
   }
 
-  private int countConnectedClients() {
-    int count = 0;
-
-    for (Server server : servers) count += server.countConnectedClients();
```

Comment:
`Server#countConnectedClients()` is dead code now. Should it be removed?

Reply:
I kept it in order to be still aligned to `Server` the Cassandra 4 & 3.11

Reply:
If it's dead code, lets remove from Stargate. We do not have to be 1:1 with C* code, I think :thinking:

Reply:
Removed